src/RipplePage.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component} from 'react';
+import ComponentPage from './ComponentPage.js';
+import HeaderBar from './HeaderBar.js';
+import {MDCRipple} from '@material/ripple';
+
+import './styles/RipplePage.scss';
+
+const RipplePage = () => {
+  return (
+    <div>
+      <HeaderBar />
+      <ComponentPage
+        hero={<RippleHero />}
+        title='Ripple'
+        description=''
```

Comment:
description:

Ripples are visual representations used to communicate the status of a component or interactive element.